homeassistant/components/tessie/sensor.py

```diff
@@ -0,0 +1,203 @@
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfPressure.BAR,
+        suggested_unit_of_measurement=UnitOfPressure.PSI,
+        device_class=SensorDeviceClass.PRESSURE,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key="vehicle_state-tpms_pressure_fr",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfPressure.BAR,
+        suggested_unit_of_measurement=UnitOfPressure.PSI,
+        device_class=SensorDeviceClass.PRESSURE,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key="vehicle_state-tpms_pressure_rl",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfPressure.BAR,
+        suggested_unit_of_measurement=UnitOfPressure.PSI,
+        device_class=SensorDeviceClass.PRESSURE,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key="vehicle_state-tpms_pressure_rr",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfPressure.BAR,
+        suggested_unit_of_measurement=UnitOfPressure.PSI,
+        device_class=SensorDeviceClass.PRESSURE,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key="climate_state-inside_temp",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key="climate_state-outside_temp",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key="climate_state-driver_temp_setting",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key="climate_state-passenger_temp_setting",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        suggested_display_precision=1,
+    ),
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the Tessie sensor platform from a config entry."""
+    coordinators = hass.data[DOMAIN][entry.entry_id]
+
+    async_add_entities(
+        [
+            TessieSensorEntity(coordinator, description)
+            for coordinator in coordinators
+            for description in DESCRIPTIONS
+            if description.key in coordinator.data
+        ]
```

Comment:
```suggestion
            TessieSensorEntity(coordinator, description)
            for coordinator in coordinators
            for description in DESCRIPTIONS
            if description.key in coordinator.data
```

This can be a generator expression.  linter will complain about indent for this suggestion